src/size_label_prs.py

```diff
@@ -20,10 +22,61 @@ def size_label_prs(data, pull):
     elif additions < 500:
         label = "Size/XL"
 
+    else:
+        label = "Size/XXL"
+        comment = "Please try to break up this PR, it is very large."
+
     if label in labels:
         return
 
     else:
         print(f"Labeling {pull.title}: {label}")
         [pull.remove_from_labels(lb) for lb in labels if lb.lower().startswith("size/")]
         pull.add_to_labels(label)
+        print(f"Commenting on {pull.title}: {comment}")
+        if comment:
+            last_commit = list(pull.get_commits())[-1]
+            pull.create_review(commit=last_commit, body=comment)
```

Comment:
create_review will not comment last by default?

Reply:
I will re-verify. Was added during an 'actions' degradation